mobile/lib/src/views/setup_action_page.dart

```diff
@@ -86,7 +86,7 @@ class _SetupActionPageState extends State<SetupActionPage> {
                 child: Align(
                     alignment: Alignment.centerLeft,
                     child: Text(
-                      "${availableActions.length} ${AppLocalizations.of(context).avalableActionsFor} ",
+                      "${availableActions == null ? 0 : availableActions!.length} ${AppLocalizations.of(context).avalableActionsFor} ",
```

Comment:
You can also use availableActions?.length ?? 0 if you prefer

Reply:
You can't actually. However, in Dart, you can define a value to use if a variable is null in a ternary. But you can't access a property